src/xtgeo/grid3d/_gridprop_lowlevel.py

```diff
@@ -93,10 +113,9 @@ def update_carray(self, undef=None, discrete=None, dtype=None, order="F"):
         values = values.astype(dtype)
 
     values = ma.filled(values, undef)
+    values = np.asfortranarray(values)
 
-    if order == "F":
-        values = np.asfortranarray(values)
-        values1d = np.ravel(values, order="F")
+    values1d = np.ravel(values, order=order)
```

Comment:
bug? values1d was only defined if order = "F".

Reply:
Nice catch. Does it matter if `np.asfortranarray(values)` is called if not invokved with "F"?

Reply:
Unsure, reverted so only does `asfortranarray(...)` if order is F.